Source/Core/Resource.js

```diff
@@ -827,10 +872,13 @@ define([
 
     /**
      * Asynchronously loads the given image resource.  Returns a promise that will resolve to
-     * an {@link Image} once loaded, or reject if the image failed to load.
+     * an {@link https://developer.mozilla.org/en-US/docs/Web/API/ImageBitmap|ImageBitmap} if the browser supports `createImageBitmap` or otherwise an
+     * {@link https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement|Image} once loaded, or reject if the image failed to load.
      *
-     * @param {Boolean} [preferBlob = false]  If true, we will load the image via a blob.
-     * @returns {Promise.<Image>|undefined} a promise that will resolve to the requested data when loaded. Returns undefined if <code>request.throttle</code> is true and the request does not have high enough priority.
+     * @param {String|Object} options An object with the following properties.
+     * @param {Boolean} [options.preferBlob=false] If true, we will load the image via a blob.
+     * @param {Boolean} [options.flipY=true] If true, image will be vertially flipped during decode. Only applies if the browser supports `createImageBitmap`.
+     * @returns {Promise.<Image>|Promise.<ImageBitmap>|undefined} a promise that will resolve to the requested data when loaded. Returns undefined if <code>request.throttle</code> is true and the request does not have high enough priority.
```

Comment:
What is the functional difference between `Image` and `ImageBitmap`?  Is this technically a breaking change but doesn't matter in practice?  Or will this break user code?  If all existing code will continue to work, great; but if it's a breaking change then we should note that.

For example, I just found out about https://developer.mozilla.org/en-US/docs/Web/API/ImageBitmap/close, it doesn't look like we are calling it anywhere.  Do we need to handle manually releasing the ImageBitmap?

Reply:
This is technically a breaking change, yes. An example of this is actually our own code in `TextureAtlasSpec`. While you can do the same things with `Image` and `ImageBitmap`, there are some minor differences, like `ImageBitmap` doesn't have a `src` property. 

I contemplated adding an option to say `disableImageBitmap` or something but I can't think of any good reason why you'd want this in the real world. I will update `CHANGES.md` to mention it's a breaking change. 

As for `ImageBitmap.close`, I just did some quick research on this and can confirm that the memory is released when it's garbage collected. Looks like calling `close()` will allow us to explicitly control when it's released. However this will break cases where one source image is used to create multiple textures. Right now this mostly breaks specs. But I haven't been able to show that this would have a noticeable enough impact to add in this PR right now.